db/legacy_seeds/dummy_structures.rb

```diff
@@ -238,15 +238,11 @@
   # The tokens below have different unhashed version to avoid worrying about clever cryptographic attacks
   if Rails.env.review?
     EngageAndLearnApiToken.find_or_create_by!(hashed_token: "dfce9a34c6f982e8adb4b903f8b6064682e6ad1f7858c41ed8a0a7468abc8896")
-    NPQRegistrationApiToken.find_or_create_by!(hashed_token: "1dae3836ed90df4b796eff1f4a4713247ac5bc8a00352ea46eee621d74cd4fcf")
```

Comment:
maybe we can also delete `NPQRegistrationApiToken` class as well? let me know what you think